frontend/src/components/BuildInfo.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+
+function BuildInfo() {
+  const [buildInfo, setBuildInfo] = useState(null);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    fetch('/api/admin/buildInfo')
```

Comment:
doing it this way is not the end of the world but did you explore inserting them directly into the build source with the REACT_APP_ prefix? I would think that is doable

Reply:
While that might work for when it's in a deployed environment, it will not work for when it running local and need to leverage git to identify the branch information.